build/media-suite_compile.sh

```diff
@@ -1113,6 +1114,7 @@ _check=(bin-global/{{c,d}j,benchmark_}xl.exe)
 if [[ $jpegxl = y ]] && do_vcs "https://gitlab.com/wg1/jpeg-xl.git"; then
     do_uninstall "${_check[@]}"
     do_pacman_install lcms2
+    do_pacman_install libjpeg-turbo
```

Comment:
libjpeg-turbo I think is a dependency of lcms2, or vice-versa. No need for both.
Or if both are indeed needed you can just call do_pacman_install once with both parameters.

Reply:
I don't know what is the creator's vision. They see some libraries are in pacman. Other libraries such as libtiff, libpng aren't.

Reply:
that might be related to a pkgconf bug I think

Reply:
Why use lcms2 if there is google skcms in c++11?

Reply:
Because mpv already uses lcms2.
Collapse these two into the same line. They're not dependencies.

Reply:
I understand. skcms is not included in mpv, there is no need to use the library.